.travis.yml

```diff
@@ -1,39 +1,32 @@
----
 language: node_js
 
 sudo: false
 
+matrix:
+  include:
+    - node_js: 0.12
```

Comment:
It's fine too, but I don't think it matters much as long as they can run. Different versions of node don't produce different outputs from these hopefully.